web/web.go

```diff
@@ -127,6 +128,8 @@ func New(o *Options) *Handler {
 		return o.Context, nil
 	})
 
+	cwd, _ := os.Getwd()
```

Comment:
We shouldn't drop the error here. We can set `cwd = "<error retrieving working directory>"` and log the error.